presets/input-history-windows/input-history-windows.html

```diff
@@ -93,6 +106,59 @@
       <!-- Get Icons from FontAwesome -->
       <!-- Copy SVG Code -->
 
+      <!-- #region Svg sources used multiple times. Automatically hidden -->
+      <!-- Arrow Left -->
+      <svg xmlns="http://www.w3.org/2000/svg">
+        <symbol
+          id="icon-arrow"
+          viewBox="0 0 512 512"
+        >
+          <!--! Font Awesome Pro 6.0.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
+          <path
+            d="M512 256C512 273.7 497.7 288 480 288H160.1l0 72c0 9.547-5.66 18.19-14.42 22c-8.754 3.812-18.95 2.077-25.94-4.407l-112.1-104c-10.24-9.5-10.24-25.69 0-35.19l112.1-104c6.992-6.484 17.18-8.218 25.94-4.406C154.4 133.8 160.1 142.5 160.1 151.1L160.1 224H480C497.7 224 512 238.3 512 256z"
+          />
+        </symbol>
+      </svg>
+
+      <!-- Right-down turn arrow -->
+      <svg xmlns="http://www.w3.org/2000/svg">
+        <symbol
+          id="icon-right-down-turn"
+          viewBox="0 0 320 512"
+        >
+          <!-- Font Awesome Pro 5.15.4 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) -->
+          <path
+            d="M313.553 392.331L209.587 504.334c-9.485 10.214-25.676 10.229-35.174 0L70.438 392.331C56.232 377.031 67.062 352 88.025 352H152V80H68.024a11.996 11.996 0 0 1-8.485-3.515l-56-56C-4.021 12.926 1.333 0 12.024 0H208c13.255 0 24 10.745 24 24v328h63.966c20.878 0 31.851 24.969 17.587 40.331z"
+          />
+        </symbol>
+      </svg>
+
+      <!-- Mouse -->
+      <svg xmlns="http://www.w3.org/2000/svg">
+        <symbol
+          id="icon-mouse"
+          viewBox="0 0 384 512"
+        >
+          <!-- Font Awesome Pro 5.15.4 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) -->
+          <path
+            d="M0 352a160 160 0 0 0 160 160h64a160 160 0 0 0 160-160V224H0zM176 0h-16A160 160 0 0 0 0 160v32h176zm48 0h-16v192h176v-32A160 160 0 0 0 224 0z"
+          />
+      </svg>
+
+      <!-- Mouse wheel left -->
+      <svg xmlns="http://www.w3.org/2000/svg">
+        <symbol
+          id="icon-wheel-left"
+          viewBox="0 0 512 512"
+        >
+          <!-- Font Awesome Pro 5.15.4 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) -->
+          <path
+            d="M256 504C119 504 8 393 8 256S119 8 256 8s248 111 248 248-111 248-248 248zM142.1 273l135.5 135.5c9.4 9.4 24.6 9.4 33.9 0l17-17c9.4-9.4 9.4-24.6 0-33.9L226.9 256l101.6-101.6c9.4-9.4 9.4-24.6 0-33.9l-17-17c-9.4-9.4-24.6-9.4-33.9 0L142.1 239c-9.4 9.4-9.4 24.6 0 34z"
+          />
+        </symbol>
```

Comment:
I've tried to adjust the icon to combine all states into one, so maybe we can use those instead:
[computer-mouse-solid.zip](https://github.com/univrsal/input-overlay/files/11189599/computer-mouse-solid.zip)

Reply:
There's NUM ENTER too, which I don't see a way to have just an icon.

For the mouse icons, isn't it better to have a single, reusable source, for the parts that repeat? That would make it easier to replace, let's say, just the mouse icon, without changing the arrows. And it's also less clutter on the inlined SVG definitions.

Let me know your thoughts. I'm curious as to why you prefer merging them.

Reply:
I fixed the alignment and icon display bugs and was trying to apply the icons you sent. But they had two colors to highlight the pushed buttons against the base icon background, and since the user can customize icon colors, they get overridden and all icons look the same.

So I tried playing with a few possibilities to punch out the buttons and, though it works, when very small it's hard to read the outline. Or it's difficult to read whether the hole on one side means the clicked button or the non-clicked one.

So I started another one from scratch, trying to make something that was both easy to quickly identify and readable when small. What do you think about these? Each is a single color with the white part punched out.
![mouse-icons](https://user-images.githubusercontent.com/1404184/232259977-a4988bd2-dc12-40f4-87a0-c869fdcc81a9.png)

I haven't applied them to the html yet. Wanted your approval first.

Reply:
The look great, thank you for your effort.